backend/endpoints/messages.py

```diff
@@ -1,3 +1,78 @@
 from fastapi import APIRouter
+from fastapi import APIRouter
+from fastapi import APIRouter, BackgroundTasks, HTTPException, status
+from schema.message import Message, ThreadRequest
+from schema.response import ResponseModel
+from starlette.responses import JSONResponse
+from utils.centrifugo import Events, centrifugo_client
+from utils.db import DataStorage
+from utils.room_utils import get_room
```

Comment:
`get_room` imported but unused. You can remove it.
`git commit `would not even pass with the pre-commit config. @AI-fae 😒😒

Reply:
For some reason I am unaware of, the pre-commit config didn't work for me.
Was wondering if someone took it off or something.

Thanks for the review. 
I have effected the changes.